### PR TITLE
Modern cicd - manually merged

### PR DESCRIPTION
**Pull request by stevemcghee**
due to the madness following the tf merge, here's an attempt to pull in shobit's work in #44 
I previously attempted in #57 but it was bonkers.

### CODE REVIEW

**Comment by stevemcghee** [comment]
this works now.  checking one last time on a clean project end-to-end. 
(* apart from SLOs)

**Comment by stevemcghee** [comment]
ahoy @muncus it works

**Comment by stevemcghee** [comment]
note: there are two instances where I had to hardcode a branch, see the last couple commits.  these should be removed on submission. (or just after).

File: modules/onboard-app/outputs.tf [hunk 1]
```diff
@@ -0,0 +1,12 @@
+//output "pipeline_id" {
```
Comment (bgood) [inline modules/onboard-app/outputs.tf:1]: Everything in this file is commented out.... can delete?

**Review by bgood — COMMENTED** [review]
All in all it looks pretty good.  The onboarding flow seems complete.

There are files with large chunks of code commented out.  Might be worth cleaning that up.  

Also diagrams would be helpful to make the flows more apparent and easier to follow.

File: modules/onboard-app/outputs.tf (hunk 1 above)
Comment (stevemcghee) [reply modules/onboard-app/outputs.tf:1]: yup deleting.

**Comment by stevemcghee** [comment]
works end to end now.

**Review by muncus — APPROVED** [review]
whoof. good job on the merge here, that was messy!

**Pull request merged**